extensions/wikia/ArticleVideo/ArticleVideoContext.class.php

```diff
@@ -45,28 +45,26 @@ public static function getFeaturedVideoData( string $pageId ) {
 				true
 			);
 
-			if ( !empty( $details ) ) {
+			if ( !empty( $details ) && !empty( $videoData['playlist'] ) && empty( $videoData['playlist'][0] ) ) {
```

Comment:
[want to test it? 😉](https://wikia-inc.atlassian.net/wiki/spaces/~mszabo/blog/2018/02/02/226721977/Testing+external+HTTP+clients+in+MediaWiki)

Reply:
Sure - I'll run that by the team and we'll figure out if it can help us. Definitely worth considering!